lib/prod-server/app.js

```diff
@@ -53,15 +53,21 @@ module.exports = ({
   }
   server.use((req, res) => {
     const url = req.url.replace(baseMatch, '/')
-    runner(url).then(({ result, redirect }) => {
+    runner(url).then(({ result, redirect, options = {} }) => {
       if (redirect) {
         res.statusCode = 301
         res.setHeader('Location', redirect)
         res.setHeader('Cache-Control', 'no-cache')
         res.end()
       } else {
         res.setHeader('Content-Type', 'text/html')
-        res.statusCode = 200
+
+        const { headers = [] } = options
```

Comment:
You can have this as an object and iterate using `entries`.

```
const headers = { 'Content-type': 'application/json' }
entries(headers).forEach(([key, value]) => {
  // stuff
})
```

Making a for a slightly nicer API.

I _think_ lodash even supplies a forEach for objects which cuts out the entries middleman.

Reply:
Oo, really good idea.